resources/js/app.js

```diff
@@ -1,2 +1,3 @@
-import 'Vendor/rapidez/core/resources/js/app.js'
-import 'Vendor/rapidez/account/resources/js/callbacks.js'
\ No newline at end of file
+import "Vendor/rapidez/core/resources/js/app.js";
+import "Vendor/rapidez/account/resources/js/callbacks.js";
+import "Vendor/rapidez/wishlist/resources/js/wishlist.js";
```

Comment:
Can we find something for this to only be imported when available? The includes in the templates are wrapped within an if-statement but when we remove the wishlist package we'll get an error on this while compiling.